src/pages/index.js

```diff
@@ -123,7 +135,119 @@ function handleFormButton() {
   profileEditPopup.open();
 }
 
-function handleProfileSubmit({ title, description }) {
-  user.setUserInfo({ title, description });
+function handleProfileSubmit({ name, about }) {
+  profileEditPopup.saving(true);
+  api
+    .updateInfo(name, about)
+    .then((data) => {
+      user.setUserInfo(data);
+    })
+    .catch((err) => {
+      console.log(err);
+    })
+    .finally(() => profileEditPopup.saving(false));
   profileEditPopup.close();
 }
+
+/* -------------------------------- Api Class ------------------------------- */
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "c6c362c5-eda0-4373-9004-84a264f04392",
+    "Content-Type": "application/json",
+  },
+});
+
+//Upload initial cards to the DOM
+api
+  .getInitialCards()
```

Comment:
Better to use `Promise.all` to load initial data

Reference: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all